tests/unittests/chronikService.test.ts

```
@@ -1269,3 +1270,251 @@ describe('Additional behavior and integration tests', () => {
+        networkId: 1,
+        syncing: false,
+        lastSynced: new Date().toISOString()
+      }
+    ])
+
+    // never hit real payment layer in these tests
+    jest.spyOn(client as any, 'handleUpdateClientPaymentStatus').mockResolvedValue(undefined)
+  })
+
+  it('handles TX_REMOVED_FROM_MEMPOOL → deletes unconfirmed txs', async () => {
+    const { fetchUnconfirmedTransactions, deleteTransactions } = require('../../services/transactionService')
```

ESLint

[error] Require statement not part of import statement.

(@typescript-eslint/no-var-requires)